packages/@stylexjs/babel-plugin/__tests__/legacy/stylex-transform-legacy-shorthands-test.js

```diff
@@ -462,3 +462,475 @@ describe('Legacy-shorthand-expansion resolution', () => {
+        `,
+          { enableLogicalStylesPolyfill: false },
+        ),
+      ).toMatchInlineSnapshot(`
+        "import _inject from "@stylexjs/stylex/lib/stylex-inject";
+        var _inject2 = _inject;
+        import stylex from 'stylex';
+        _inject2(".x123j3cw{padding-top:5px}", 4000);
+        _inject2(".xs9asl8{padding-bottom:5px}", 4000);
+        _inject2(".xaso8d8{padding-inline-start:5px}", 3000);
+        _inject2(".x2vl965{padding-inline-end:10px}", 3000);
+        _inject2(".x1nn3v0j{padding-top:2px}", 4000);
+        _inject2(".x14vy60q{padding-inline-end:2px}", 3000);
+        _inject2(".x1120s5i{padding-bottom:2px}", 4000);
+        "x1nn3v0j x14vy60q x1120s5i";"
+      `);
+    });
+
+    test('borderColor: basic shorthand', () => {
+      expect(
+        transform(
+          `
+          import stylex from 'stylex';
+          const styles = stylex.create({
+            foo: {
+              borderColor: 'red blue green yellow'
+            }
+          });
+          stylex(styles.foo);
+        `,
+          { enableLogicalStylesPolyfill: false },
+        ),
+      ).toMatchInlineSnapshot(`
+        "import _inject from "@stylexjs/stylex/lib/stylex-inject";
+        var _inject2 = _inject;
+        import stylex from 'stylex';
+        _inject2(".x1uu1fcu{border-top-color:red}", 4000);
+        _inject2(".xcejqfc{border-inline-end-color:blue}", 3000);
+        _inject2(".x1hnil3p{border-bottom-color:green}", 4000);
+        _inject2(".xqzb60q{border-inline-start-color:yellow}", 3000);
+        "x1uu1fcu xcejqfc x1hnil3p xqzb60q";"
+      `);
+    });
+
+    test('borderWidth: basic shorthand', () => {
+      expect(
+        transform(
+          `
+          import stylex from 'stylex';
+          const styles = stylex.create({
+            foo: {
+              borderWidth: '1px 2px 3px 4px'
+            }
+          });
+          stylex(styles.foo);
+        `,
+          { enableLogicalStylesPolyfill: false },
+        ),
+      ).toMatchInlineSnapshot(`
+        "import _inject from "@stylexjs/stylex/lib/stylex-inject";
+        var _inject2 = _inject;
+        import stylex from 'stylex';
+        _inject2(".x178xt8z{border-top-width:1px}", 4000);
+        _inject2(".x1alpsbp{border-inline-end-width:2px}", 3000);
+        _inject2(".x2x41l1{border-bottom-width:3px}", 4000);
+        _inject2(".x56jcm7{border-inline-start-width:4px}", 3000);
+        "x178xt8z x1alpsbp x2x41l1 x56jcm7";"
+      `);
+    });
+  });
+
+  describe('while using standard logical properties', () => {
+    test('paddingInline: basic shorthand', () => {
+      expect(
+        transform(
+          `
+          import stylex from 'stylex';
+          export const styles = stylex.create({
+            foo: {
+              paddingInline: 5
+            }
+          });
+          stylex(styles.foo);
+        `,
+          { enableLogicalStylesPolyfill: false },
+        ),
+      ).toMatchInlineSnapshot(`
```

Comment:
Tests like this don't see to be in the right place, if this file is for testing "legacy shorthands"